### PR TITLE
feat: configure demo error handler destinations

### PR DESCRIPTION
**Pull request by rsioss**

### CODE REVIEW

File: src/setup/bootstrap/createStashRecords.ts [hunk 1]
```diff
@@ -77,6 +78,34 @@ export const createSampleStashRecords = async ({
...
+          url: requiredEnvVar("DESTINATION_WEBHOOK_URL"),
+        },
+      },
+    ],
+  });
+
+  await saveErrorDestinations("destinations|errors|execution", {
+    $schema:
+      "https://raw.githubusercontent.com/Stedi-Demos/bootstrap/main/src/schemas/error-destinations.json",
+    description: "Send function execution errors to webhook",
```
Comment (BDQ) [inline src/setup/bootstrap/createStashRecords.ts:98]: I would label this as "Send delivery failures to webhook"  - its not tied to the DLQ so "function execution errors" is confusing?

**Review by BDQ — APPROVED** [review]
Left one description change suggestion

File: src/setup/bootstrap/createStashRecords.ts (hunk 1 above)
Comment (rsioss) [reply src/setup/bootstrap/createStashRecords.ts:98]: @BDQ: apologies I hadn't followed up. this PR got lost in the shuffle... I think `function execution errors` is more accurate as it encompasses things beyond just delivery failures. these destinations are essentially replacing the old `failure.json` files for failed execution tracking.

File: src/lib/saveErrorDestinations.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import {
+  DestinationErrorEvents,
+  DestinationErrorEventsSchema,
+} from "./types/Destination.js";
+
+const stash = stashClient();
+
+export const saveErrorDestinations = async (
+  id: string,
+  destinations: object
```
Comment (RossWilliams) [inline src/lib/saveErrorDestinations.ts:14]: you could set this to the DestinationErrorEvents type

**Review by RossWilliams — APPROVED** [review]

**Pull request merged**